demisto_sdk/commands/init/contribution_converter.py

```diff
@@ -234,6 +240,37 @@ def format_converted_pack(self) -> None:
             input=self.pack_dir_path, from_version=from_version, no_validate=True, update_docker=True, assume_yes=True
         )
 
+    def generate_readme_for_pack_content_item(self, yml_path: str) -> None:
+        """Runs the demisto-sdk's generate-docs command on the pack converted from the contribution zipfile"""
+        file_type = find_type(yml_path)
+        file_type = file_type.value if file_type else file_type
+        if file_type == 'integration':
+            generate_integration_doc(yml_path)
+        if file_type == 'script':
+            generate_script_doc(input=yml_path, examples=[])
+        if file_type == 'playbook':
+            generate_playbook_doc(yml_path)
+
+    def generate_reamde_for_new_content_pack(self):
+        for pack_subdir in get_child_directories(self.pack_dir_path):
+            basename = os.path.basename(pack_subdir)
+            if basename in {SCRIPTS_DIR, INTEGRATIONS_DIR}:
+                directories = get_child_directories(pack_subdir)
+                for directory in directories:
+                    files = get_child_files(directory)
+                    for file in files:
+                        file_name = os.path.basename(file)
+                        if file_name.startswith('integration') or file_name.startswith('script'):
+                            unified_file = file
+                            self.generate_readme_for_pack_content_item(unified_file)
+                            os.remove(unified_file)
+            elif basename == 'Playbooks':
+                files = get_child_files(pack_subdir)
+                for file in files:
+                    file_name = os.path.basename(file)
+                    if file_name.endswith('.yml'):
```

Comment:
In this case it's not right. 
Most of our playbooks (not test playbook) will not start with the prefix `playbook`.
That's why I'm checking only the `yml` ending.

Reply:
For contribution zip files downloaded from the server, the playbook YAMLs should have a prefix of `playbook` (unless it has been changed since I worked on it which I don't think it did).

Reply:
Checked it and you are right.